common/src/test/java/com/datastrato/gravitino/json/TestJsonUtils.java

```diff
@@ -432,4 +439,89 @@ void testIndexDTOSerde() throws JsonProcessingException {
         objectMapper.readValue(expected, IndexDTO.class),
         objectMapper.readValue(jsonValue, IndexDTO.class));
   }
+
+  @Test
+  void testFromDTOPartitionDTOWhenIdentityPartitionDTOToPartition() {
+
+    // given
+    String[] field1 = {"dt"};
+    String[] field2 = {"country"};
+    LiteralDTO literal1 =
+        new LiteralDTO.Builder().withDataType(Types.DateType.get()).withValue("2008-08-08").build();
+    LiteralDTO literal2 =
+        new LiteralDTO.Builder().withDataType(Types.StringType.get()).withValue("us").build();
+    String[][] fieldNames = {field1, field2};
+    LiteralDTO[] values = {literal1, literal2};
+
+    Map<String, String> properties = Collections.singletonMap("key", "value");
+    PartitionDTO identityPartitionDTO =
+        IdentityPartitionDTO.builder()
+            .withFieldNames(fieldNames)
+            .withName("IdentityPartition")
+            .withValues(values)
+            .withProperties(properties)
+            .build();
+    // when
+    IdentityPartition identityPartition =
+        (IdentityPartition) DTOConverters.fromDTO(identityPartitionDTO);
```

Comment:
both `fromDTO` and `toDTO` need to be tested

Reply:
@mchades added tests for toDTO and moved tests to new class